client_solaris.go

```diff
@@ -0,0 +1,20 @@
+package containerd
+
+import (
+	"fmt"
+	"net"
+	"strings"
+	"time"
+)
+
+// DefaultAddress is the default unix socket address
+const DefaultAddress = "/var/run/containerd/containerd.sock"
```

Comment:
the only difference between client_unix.go and client_solaris.go is DefaultAddress.  i could create client_address_unix.go and client_address_solaris.go, move DefaultAddress to there, and then compile client_unix.go on solaris.  lemme know if that's what you're thinking.